files/build_templates/docker_image_ctl.j2

```diff
@@ -131,6 +131,9 @@ function postStartAction()
             /usr/bin/db_migrator.py -o migrate
         fi
     fi
+    # Add redis UDS to the redis group and give read/write access to the group
+    REDIS_SOCK="/var/run/redis/redis.sock"
```

Comment:
To support multi-asic platforms .. update the REDIS_SOCK --> REDIS_SOCK=/var/run/redis$DEV/redis.sock, so that unix socket for all the database docker intsances will be covered. 

$DEV varies from '', 0, 1, 2, .. as per the number of namespaces.

Reply:
@judyjoseph multi-asic was already using UDS and is not related to the change I made to single ASIC. I changed this for single ASIC only. I am curious if bgp restart was successful before my change on multi-asic

Reply:
This is required for the change in [PR:4941](https://github.com/Azure/sonic-buildimage/pull/4941/files?diff=unified&w=1#diff-2901ad8ea8e7ba16059aa09588944b30L300) The line below is for the case when multi-asic in involved and that was already using UDS.

Reply:
Yes, for multi-asic with namespace we were already UDS .. but was using sudo to get over this issue ( https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/docker_image_ctl.j2#L357 ). If we are looking to change the redis sock group ownership .. it is better to make it similar for all UDS's in any platform single/multi